backend/api/Controllers/EchoMissionController.cs

```diff
@@ -95,4 +95,71 @@ public async Task<ActionResult<EchoMission>> GetEchoMission([FromRoute] int miss
             return new StatusCodeResult(StatusCodes.Status500InternalServerError);
         }
     }
+    [HttpGet]
+    [Route("installation/{installationCode}")]
+    [ProducesResponseType(typeof(EchoMission), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [ProducesResponseType(StatusCodes.Status502BadGateway)]
+    public async Task<ActionResult<EchoMission>> GetEchoMissionsFromInstallation([FromRoute] string installationCode)
```

Comment:
Should make this an optional query parameter to filter on the "GetMissions" endpoint instead of making it an endpoint on its own.